docroot/modules/custom/mass_content/src/Entity/Bundle/paragraph/ListItemDocumentsBundle.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\mass_content\Entity\Bundle\paragraph;
+
+final class ListItemDocumentsBundle extends ParagraphBundle {
+
+  public function getManualDescription(): string {
+    if (!$this->get('field_listitemdoc_desc_manual')->isEmpty()) {
+      return $this->field_listitemdoc_desc_manual->value;
+    }
+    return '';
+  }
```

Comment:
Our other bundle classes dont return scalars like a string. We return the ListItem and then expect the caller to use methods like isEmpty() or getValue(). Lets stick with that pattern.

Reply:
Changed